var/spack/repos/builtin/packages/coreutils/package.py

```diff
@@ -25,6 +25,8 @@ class Coreutils(AutotoolsPackage, GNUMirrorPackage):
 
     variant("gprefix", default=False, description="prefix commands with 'g', to avoid conflicts with OS utilities")
 
+    patch('coreutils-8.32-ls-restore-8.31-behavior.patch', when='@8.32 target=aarch64:')
```

Comment:
Can we use:
```suggestion
    patch(
        'https://src.fedoraproject.org/rpms/coreutils/raw/6b50cb9f/f/coreutils-8.32-ls-removed-dir.patch',
        sha256='5878894375a8fda98150783430b30c0b7104899dc5522034ebcaf8c961183b7e'
        when='@8.32 target=aarch64:'
    )
```
and remove the explicit patch file stored in Spack? Also, it would be good if you can write a very brief comment above the directive explaining what the patch does.

Reply:
will do.  didn't know patch could do this!  pretty nifty.